### PR TITLE
Remove the add of 1 day.

### PR DESCRIPTION
**Pull request by jdalton**
This PR fixes the off by 1 day issue of #33.

### CODE REVIEW

**Comment by pvorb** [comment]
@jdalton Thank you for the PR. I'll start a review right now.

File: src/charts.js [hunk 1]
```diff
@@ -162,11 +163,7 @@ function getDailyDownloadData(downloadData, dateRange) {
         var values = [];
         for (var i = 0; i < dateRange.length; i++) {
             var key = dateToDayKey(dateRange[i]);
-            var dateAsMidnight = moment(dateRange[i]).utc().add(1, 'day').set({
-                hour: 0,
-                minute: 0,
-                second: 0
-            }).toDate().getTime();
+            var dateAsMidnight = +moment(dateRange[i]).startOf('day');
```
Comment (pvorb) [inline src/charts.js]: I prefer `.valueOf()`.

**Comment by pvorb** [comment]
The new version is now available from npm-stat.com.

File: src/charts.js (hunk 1 above)
Comment (jdalton) [inline src/charts.js]: Ok cool.

**Comment by jdalton** [comment]
Awesome! Thank you!

**Pull request merged**